docs/integrations.md

```diff
@@ -72,6 +72,7 @@ I've added a ⭐ to projects or posts that have a significant following, or had
 
 ## Projects + scripts 
 
+- [ntfy-android-builder](https://github.com/TheBlusky/ntfy-android-builder) - Script for building ntfy-android with custom Firebase configuration (Docker/Shell)
```

Comment:
Well. That's quite cocky to put it at the top. Why don't you be nice and move it to the bottom :-D

Reply:
I didn't mean to be rude I (too quickly) thought upper was last updated, sorry about that. I updated it  and put it to bottom.

Reply:
:wink: